src/com/chocohead/clm/ClothesLineMotor.java

```diff
@@ -31,45 +30,40 @@
+		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "motor"), new BlockItem(MOTOR, new Settings().group(ClotheslineItemGroups.ITEMS)));
+		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "broken_motor"), new Item(new Settings().group(ClotheslineItemGroups.ITEMS) ) {
 			@Override
 			@Environment(EnvType.CLIENT)
-			public void buildTooltip(ItemStack stack, World world, List<TextComponent> tooltip, TooltipContext options) {
-				tooltip.add(new TranslatableTextComponent("item.clothesline_motor.broken_motor.disclaimer").applyFormat(TextFormat.GRAY));
+			public void appendTooltip(ItemStack stack, World world, List<Text> tooltip, TooltipContext options) {
+				tooltip.add(new TranslatableText("item.clothesline_motor.broken_motor.disclaimer").styled(style -> style.withColor(Formatting.GRAY)));
 			}
 		});
 
-		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "motor_casing"), new Item(new Settings().itemGroup(ClotheslineItemGroups.ITEMS)));
-		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "coil"), new Item(new Settings().itemGroup(ClotheslineItemGroups.ITEMS)) {
+		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "motor_casing"), new Item(new Settings().group(ClotheslineItemGroups.ITEMS)));
+		Registry.register(Registry.ITEM, new Identifier("clothesline_motor", "coil"), new Item(new Settings().group(ClotheslineItemGroups.ITEMS)) {
 			@Override
 			@Environment(EnvType.CLIENT)
-			public void buildTooltip(ItemStack stack, World world, List<TextComponent> tooltip, TooltipContext options) {
-				tooltip.add(new TranslatableTextComponent("item.clothesline_motor.coil.disclaimer").applyFormat(TextFormat.GRAY));
+			public void appendTooltip(ItemStack stack, World world, List<Text> tooltip, TooltipContext options) {
+				tooltip.add(new TranslatableText("item.clothesline_motor.coil.disclaimer").styled(style -> style.withColor(Formatting.GRAY)));
 			}
 		});
 
-		ChunkLoadCallback.EVENT.register((world, chunk, pos) -> {
-			if (world.isClient) return;
-
-			BlockState activeUpMotor = MOTOR.getDefaultState().with(MotorBlock.STATUS, Status.ON).with(MotorBlock.FACING, Direction.UP);
-			BlockState activeDownMotor = MOTOR.getDefaultState().with(MotorBlock.STATUS, Status.ON).with(MotorBlock.FACING, Direction.DOWN);
-			Mutable relativePos = new Mutable(pos.getStartX(), 0, pos.getStartZ());
+		ServerChunkEvents.CHUNK_LOAD.register((world, chunk) -> {
+			Mutable relativePos = new Mutable(chunk.getPos().getStartX(), 0, chunk.getPos().getStartZ());
 
 			for (ChunkSection section : chunk.getSectionArray()) {
-				if (!ChunkSection.isEmpty(section) && (section.method_19523(activeUpMotor) || section.method_19523(activeDownMotor))) {
+				if (!ChunkSection.isEmpty(section) && (section.method_19523(state -> state.get(MotorBlock.STATUS) == Status.ON))) {
```

Comment:
Would this not need to check the state is a motor block first? Thought it got upset if you query a state for a property it doesn't have?